I have proposed this change because it solves this error (installing several core modules). Sorted is something that is already being used in `other base_user_role` tests.

```
FAIL: TestUserRole.test_group_groups_into_role
Traceback (most recent call last):
  File "/opt/odoo/auto/addons/base_user_role/tests/test_user_role.py", line 272, in test_group_groups_into_role
    self.assertEqual(new_role.implied_ids.ids, user_group_ids)
AssertionError: Lists differ: [12, 8, 80, 77, 28, 82, 32, 35, 17, 19, 63, 53, 3[121 chars], 78] != [12, 8, 38, 82, 80, 32, 77, 17, 19, 63, 28, 53, 3[121 chars], 78]
First differing element 2:
80
38
```

--- a/base_user_role/tests/test_user_role.py
+++ b/base_user_role/tests/test_user_role.py
@@ -269,4 +269,4 @@ class TestUserRole(TransactionCase):
         new_role = self.role_model.search([("name", "=", "Test Role")])
         self.assertTrue(new_role)
         # Check that the role has the correct groups
-        self.assertEqual(new_role.implied_ids.ids, user_group_ids)
+        self.assertEqual(set(new_role.implied_ids.ids), set(user_group_ids))

